_docs/performance-tuning/drill-metastore/010-using-drill-metastore.md

```diff
@@ -250,6 +282,9 @@ A table can be divided into directories, called "partitions". The `PARTITIONS` t
  - Applies to tables stored as Parquet files and only when stored in the `DFS` storage plugin.
  - Disabled by default. You must enable this feature through the `metastore.enabled` system/session option.
 
+### Limitations of the 1.18 release
+ - Applies to all file system storage plugin formats except for MaprDB.
```

Comment:
Again, since Drill 1.17 is released, should we list Drill 1.17 limitations as well?

Reply:
The previous section contains limitations for Drill 1.17.